perdiem/urls.py

```diff
@@ -88,6 +88,11 @@
         TemplateView.as_view(template_name="extra/privacy.html"),
         name="privacy",
     ),
+    url(
+        r"^investor-resources/?$",
```

Comment:
You could make this something like `r"^resources/investor/?$"` and then change the artist resources to be something like `r"^resources/artist/?$"`.

Reply:
Yeah, I think this makes sense. Just realized that some of my base and home updates include these links and are updated in another branch. So I'll just submit another PR later with these changes